src/main/resources/hudson/plugins/ec2/EC2Cloud/config-entries.jelly

```diff
@@ -52,6 +52,9 @@ THE SOFTWARE.
     <f:entry title="${%Session Name}" field="roleSessionName">
       <f:textbox checkDependsOn="roleArn"/>
     </f:entry>
+    <f:entry title="${%Clean Up Orphan Nodes}" field="cleanUpOrphanedNodes">
```

Comment:
You need to create a `help-cleanUpOrphanedNodes.html`, or nobody is going to know what this means or whether they should use it.